murageden/we-connect-v2-react-front-end
# [#158820636] Display User Friendly Messages

#### What does this PR do?
Makes toast messages more human readable
#### Description of Task to be completed?
Make messages returned on user actions more readable, with less jargon like business ids
#### How should this be manually tested?
* Set up the application as described on the `README`
* Switch to branch: `ft-user-friendly-messages-158820636`
* Access the development web app at localhost
* On login, register business, update business or adding review, the message displayed on the top of the page should be clear and readable
#### Questions:
How can this be improved?
